modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/service_base_impl.ftl

```diff
@@ -243,6 +243,100 @@ import org.osgi.service.component.annotations.Reference;
 			return ${entity.variableName}Persistence.update(${entity.variableName});
 		}
 
+		<#if entity.hasExternalReferenceCode() && entity.hasLazyReference() && entity.hasEntityColumn("companyId") && stringUtil.equals(entity.name, "User") && entity.hasEntityColumn("batchImportStatus")>
+			/**
+			 * Adds the ${entity.humanName} to the database. Also notifies the appropriate model listeners.
```

Comment:
Probably we should add a more specific comment to specify that is for the temporary data because we set the batchImportStatus to 1

Reply:
Yes, we should do it.